htdocs/product/fournisseurs.php

```diff
@@ -723,6 +758,24 @@ function update_price_from_multicurrency() {
     				print '</tr>';
 				}
 
+				$extralabels=$extrafields->fetch_name_optionals_label("product_fournisseur_price");
+				// Extrafields
+				$resql = $db->query("SELECT * FROM " . MAIN_DB_PREFIX . "product_fournisseur_price_extrafields WHERE fk_object = " . $rowid);
+				if ($db->num_rows($resql) != 1) {
+					foreach ($extralabels as $key => $value) {
```

Comment:
$extralabels is deprecated.
You must use $extrafields->attributes["product_fournisseur_price"]['label'] instead.